cmd/launcher/launcher.go

```diff
@@ -74,7 +74,7 @@ const (
 	desktopMenuSubsystemName = "kolide_desktop_menu"
 	authTokensSubsystemName  = "auth_tokens"
 	katcSubsystemName        = "katc_config" // Kolide ATC
-	ztaInfoSubsystemName     = "zta_info"
+	dt4aInfoSubsystemName    = "zta_info"
```

Comment:
For an easier transition between the old and new subsystem names, could we support two subsystems and have them handled by the same consumer and store? i.e. something like --

```golang
ztaInfoSubsystemName     = "zta_info"
dt4aInfoSubsystemName    = "dt4a_info"

// ...

// Set up consumer to receive DT4A info from the control server in both current and legacy subsystem
dt4aInfoConsumer := keyvalueconsumer.NewConfigConsumer(k.Dt4aInfoStore())
if err := controlService.RegisterConsumer(ztaInfoSubsystemName, dt4aInfoConsumer); err != nil {
	return fmt.Errorf("failed to register ZTA info consumer: %w", err)
}
if err := controlService.RegisterConsumer(dt4aInfoSubsystemName, dt4aInfoConsumer); err != nil {
	return fmt.Errorf("failed to register dt4a info consumer: %w", err)
}
```

Reply:
sure, that makes sense, done